commons-vfs2/src/main/java/org/apache/commons/vfs2/provider/ftp/FtpClient.java

```diff
@@ -69,5 +69,8 @@ default void setBufferSize(final int bufferSize) throws FileSystemException {
     }
 
     OutputStream storeFileStream(String relPath) throws IOException;
-
+    
+    boolean features() throws IOException;

```

Comment:
Hi @moikeygraham,
- Thank you your PR.
- You MUST use default methods to provide binary backward compatibility.
- Please provide Javadocs for new methods.

Reply:
@garydgregory thanks for the prompt review! 

I have made the changes you suggested in latest commit.